src/util/taproot.rs

```diff
@@ -756,20 +760,54 @@ impl ControlBlock {
     }
 }
 
+/// Inner type representing future (non-tapscript) leaf versions. See [`LeafVersion::Future`].
+///
+/// NB: NO PUBLIC CONSTRUCTOR!
+/// The only way to construct this is by converting u8 to LeafVersion and then extracting it.
```

Comment:
TODO: create good first issue for rewriting to `` `u8` `` and ``[`LeafVersion`]``

Reply:
Sorry, I am not following what do you mean here

Reply:
Putting backticks and square brackets around them :)